[assets] dragging assets folder throws errors

```server.ts:101 PUT https://utopia.pizza/v1/asset/a3e0db4b/src/assets?old_file_name=/assets 500 react_devtools_backend.js:2557 Error sending updates to VS Code Error: FS Error: {"code":"ENOENT","path":"/utopia/src/assets/public/assets/public/"}```

--- a/editor/src/components/editor/actions/actions.ts
+++ b/editor/src/components/editor/actions/actions.ts
@@ -501,7 +501,7 @@ import Meta from 'antd/lib/card/Meta'
 import utils from '../../../utils/utils'
 import { defaultConfig } from 'utopia-vscode-common'
 import { getTargetParentForPaste } from '../../../utils/clipboard'
-import { absolutePathFromRelativePath } from '../../../utils/path-utils'
+import { absolutePathFromRelativePath, stripLeadingSlash } from '../../../utils/path-utils'
 import { resolveModule } from '../../../core/es-modules/package-manager/module-resolution'
 
 function applyUpdateToJSXElement(
@@ -3228,7 +3228,7 @@ export const UPDATE_FNS = {
         if (oldContent != null && (isImageFile(oldContent) || isAssetFile(oldContent))) {
           // Update assets.
           if (isLoggedIn(userState.loginState) && editor.id != null) {
-            updateAssetFileName(editor.id, action.oldPath, action.newPath)
+            updateAssetFileName(editor.id, stripLeadingSlash(oldPath), newPath)
           }
         }
       })

